src/python/WMCore/MicroService/MSTransferor/MSTransferor.py

```diff
@@ -499,7 +518,16 @@ def makeTransferRucio(self, wflow, dataIn, dids, dataSize, grouping, copies, nod
             self.logger.info("Creating rule for workflow %s with %d DIDs in container %s, RSEs: %s, grouping: %s",
                              wflow.getName(), len(dids), dataIn['name'], rseExpr, grouping)
             try:
-                res = self.rucio.createReplicationRule(dids, rseExpr, **ruleAttrs)
+                # make decision about current workflow, if it is new request we'll create
+                # new replication rule, otherwise we'll move replication rule
+                if wflow.dataReplacement:
+                    rids = self.getRuleIdsFromDoc(wflow.getName())
+                    for rid in rids:
+                        # the moveReplcationRule will raise different exceptions
```

Comment:
Typo in this line.
And what do you mean with "operation is not normal"? If the server returns an error?

Reply:
fixed and clarified the comment